### ISSUE TITLE
Slices of same song in both training and test sets?

### ISSUE THREAD

**Issue by voodoohop** [despoisj/DeepAudioClassification#15]
I have the impression the classification is getting such good results because the test set contains slices of the same songs that are in the training set.

I get that the test set has different slices than the training set but if we are training on different slices of the same song the neural network could be just learning to identify songs instead of genres.

**Comment by despoisj** [comment]
That might indeed be an issue. Try changing the selection of slices in createDatasetFromSlices (if the information of the original song from which the slice is extracted is still there) to remove this bias!

**Comment by lnris2** [comment]
Sorry to hijack your problem, but you know where i can get a working Sox version?, i just installed and can convert to mp3 but looks like this don`t recognize some commands.

**Comment by despoisj** [comment]
I'm sorry, I don't remember at all where I got Sox from on my mac. The only
thing I remember is that it was a real struggle! Good luck on that! :/

2017-11-17 0:26 GMT+01:00 lnris2 <notifications@github.com>:

**Comment by panamantis** [comment]
sox 14-4-2 worked for me on windows
sox 14-4-0 worked on linux
